app/controllers/account/transactions_controller.rb

```diff
@@ -12,8 +12,7 @@ def index
   end
 
   def update
-    @entry.update!(entry_params.merge(amount: amount))
-    @entry.sync_account_later
```

Comment:
Removed the `sync_later` because this triggers a refresh of the entire page (via the Sync broadcasts) which gets annoying for updating transaction details.  This will cause out-of-sync data if the user changes the amount or date, but will be updated on the next sync.

Eventually we'll want to get more granular with _when_ we trigger the re-sync, but for now this feels like a decent tradeoff.